src/locale/locales/fr/messages.po

```diff
@@ -5217,7 +5208,7 @@ msgstr "Cet appel sera envoyé à <0>{0}</0>."
 
 #: src/screens/Messages/Conversation/ChatDisabled.tsx:104
 msgid "This appeal will be sent to Bluesky's moderation service."
-msgstr ""
+msgstr "Cet appel sera envoyé aux services de modération de Bluesky."
```

Comment:
```suggestion
msgstr "Cet appel sera envoyé au service de modération de Bluesky."
```
I think `service de modération` should be singular rather than plural here?